ReflectometryServer/beamline.py

```diff
@@ -1,6 +1,8 @@
 """
 Resources at a beamline level
 """
+from builtins import str
```

Comment:
I do not think this is required as we are not making it backwards compatible with python 2.